client/login/controller.js

```diff
@@ -12,15 +12,16 @@ import HandleEmailedLinkForm from './magic-login/handle-emailed-link-form';
 
 export default {
 	login( context, next ) {
-		const { lang, path, params: { flow, twoFactorAuthType } } = context;
+		const { lang, path, params: { flow, twoFactorAuthType }, query } = context;
 
 		context.primary = (
 			<WPLogin
 				locale={ lang }
 				path={ path }
 				twoFactorAuthType={ twoFactorAuthType }
 				socialConnect={ flow === 'social-connect' }
-				privateSite={ flow === 'private-site' } />
+				privateSite={ flow === 'private-site' }
+				clientId={ query.clientId } />
```

Comment:
Historically, WP.com has used lowercase, and underscores to separate words. Ie, `client_id`.